client/src/ts/trainees/InternalProject.tsx

```diff
@@ -0,0 +1,330 @@
+      });
+      */
+
+    // Clean-up function
+    return () => {
+      mounted = false;
+    };
+  };
+
+  /**
+   * Returns the interface
+   */
+  return (
+    <div>
+      <InternalProjectInformationDialog
+        name={name}
+        kuerzel={kuerzel}
+        traineegeneration={traineegeneration}
+        kickoff={kickoff}
+        angebotAbgegeben={angebotAbgegeben}
+        apAbgegeben={apAbgegeben}
+        apDatum={apDatum}
+        zpDatum={zpDatum}
+        projektmitglieder={projektmitglieder}
+        dlAbgegeben={dlAbgegeben}
+        zpAbgegeben={zpAbgegeben}
+        qualitaetsmanager={qualitaetsmanager}
+        internalProjectInfoDialogOpen={internalProjectInfoDialogOpen}
+        handleInternalProjectInfoDialogClose={handleInternalProjectInfoDialogClose}
+        setName={setName}
+        setKuerzel={setKuerzel}
+        setTraineegeneration={setTraineegeneration}
+        setKickoff={setKickoff}
+        setAngebotAbgegeben={setAngebotAbgegeben}
+        setApDatum={setApDatum}
+        setApAbgegeben={setApAbgegeben}
+        setZpDatum={setZpDatum}
+        setZpAbgegeben={setZpAbgegeben}
+        setDlAbgegeben={setDlAbgegeben}
+        setProjektmitglieder={setProjektmitglieder}
+        setQualitaetsmanager={setQualitaetsmanager}
+        updateInternalProjectDetails={updateInternalProjectDetails}
+      />
+
+      <div className="content-page">
+        <Paper className={classes.paper}>
+          <div className={classes.categoryHeader}>
+            <div>
+              <Typography variant="h6">
+                <strong>Informationen zum internen Projekt</strong>
+              </Typography>
+            </div>
+            <div>
+              {doesPermissionsHaveSomeOf(auth.permissions, [15]) ? (
+                <IconButton onClick={(event) => handleInternalProjectInfoDialogOpen(event)}>
+                  <Edit fontSize="inherit" />
+                </IconButton>
+              ) : null}
+            </div>
+          </div>
+          <br></br>
+          <div className={classes.category}>
+            <InternalProjectDetail name={"Name"} value={name} />
+            <InternalProjectDetail name={"Kürzel"} value={kuerzel} />
+            <InternalProjectDetail name={"Traineegeneration"} value={traineegeneration} />
+            <InternalProjectDetail name={"Kick-Off"} value={kickoff} />
+            <InternalProjectDetail name={"Angebot abgegeben"} value={angebotAbgegeben} />
+            <InternalProjectDetail name={"AP abgegeben"} value={apAbgegeben} />
+            <InternalProjectDetail name={"AP Datum"} value={apDatum} />
+            <InternalProjectDetail name={"ZP abgegeben"} value={zpAbgegeben} />
+            <InternalProjectDetail name={"DL abgegeben"} value={dlAbgegeben} />
+            <InternalProjectDetail name={"Projektmitglieder"} value={projektmitglieder} />
+            <InternalProjectDetail name={"Qualitätsmanager"} value={qualitaetsmanager} />
+          </div>
+          <br></br>
```

Comment:
`</br>` is an invalid html element because br is self closing.
Use `<br>` or `<br/>`.

```suggestion
          <br/>